functions/billing/index.js

```diff
@@ -57,9 +57,22 @@ exports.stopBilling = async (pubsubEvent, context) => {
   if (pubsubData.costAmount <= pubsubData.budgetAmount) {
     return `No action necessary. (Current cost: ${pubsubData.costAmount})`;
   }
+  
+  if (!PROJECT_ID) {
+    return `No project specified`;
+  }
 
   await _setAuthCredential();
-  if (await _isBillingEnabled(PROJECT_NAME)) {
+  let billingEnabled;
+  try {
+    billingEnabled = await _isBillingEnabled(PROJECT_NAME);
+  }
+  catch (e) {
+    billingEnabled = true;
+    console.log('Unable to determine if billing is enabled on specified project, assuming true');
```

Comment:
Nit: `assuming true` = awkward phrasing?

Reply:
Updating to "Unable to determine if billing is enabled on specified project, assuming billing is enabled"